containerfiles/unstable

```diff
@@ -171,20 +201,6 @@ COPY rootfs/etc/xdg/weston/weston.ini							/etc/xdg/weston/
 # Default dev session configuration (with screen rotation)
 COPY rootfs/etc/xdg/weston/weston-rotated.ini						/etc/xdg/weston/
 
-# Package repositories
```

Comment:
These copy operations are superfluous, we do an earlier copy operation which is needed to have repos ready for package install operations.